Tests/chocolateyguicli.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+Import-Module helpers/gui-helpers

```

Comment:
Since there is no mention to add the directory to `PSModulePath` this import will fail for most users (myself included).

A vagrant image similar to the one used in https://github.com/chocolatey/choco should probably be used so the environment is correctly set up as expected.

Reply:
I do like the idea of the vagrant image, just didn't want to spend the cycles setting one up with all that's required to actually build the package. I've updated this line to specify the psm1 file which should solve the error that @gep13 was seeing.

Perhaps we leave the issue open with a note to add a vagrantfile in the future for this 🤔 (or maybe an invoke script that builds on the system and then does the vagrant things to reduce the time to get the vagrant running)